packages/zosfiles/__tests__/__system__/api/methods/download/Download.system.test.ts

```diff
@@ -281,6 +320,45 @@ describe("Download Data Set", () => {
                 expect(fileContents).toEqual(data);
             });
 
+            it("should download a data set and create folders and file in original letter case", async () => {
+                let error;
+                let response: IZosFilesResponse;
+
+                // TODO - convert to UPLOAD APIs when available
+                // upload data to the newly created data set
+                const data: string = "abcdefghijklmnopqrstuvwxyz";
+                const endpoint: string = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_DS_FILES + "/" + dsname + "(member)";
+                const rc = await ZosmfRestClient.putExpectString(REAL_SESSION, endpoint, [], data);
+
+                try {
+                    response = await Download.allMembers(REAL_SESSION, dsname, { preserveOriginalLetterCase: true });
+                    Imperative.console.info("Response: " + inspect(response));
+                } catch (err) {
+                    error = err;
+                    Imperative.console.info("Error: " + inspect(error));
+                }
+                expect(error).toBeFalsy();
+                expect(response).toBeTruthy();
+                expect(response.success).toBeTruthy();
+                expect(response.commandResponse).toContain(
+                    ZosFilesMessages.datasetDownloadedSuccessfully.message.substring(0, "Data set downloaded successfully".length + 1));
+
+                // convert the data set name to use as a path/file
+                const regex = /\./gi;
+                file = dsname.replace(regex, "/") + "/MEMBER.txt";
```

Comment:
Since the only difference between the tests added seems to be this line, Should we try to combine the two?
I'm thinking purely about maintenance overhead of having duplication of tests in case we had to go back and refactor something.

Reply:
The one tests on Download.dataSet() while the other tests on Download.allMembers(). Since both parts have changes, I think it makes sense to separate into two tests.